diff --git a/node_modules/react-native-fast-opencv/android/build.gradle b/node_modules/react-native-fast-opencv/android/build.gradle
index 1234567..7654321 100644
--- a/node_modules/react-native-fast-opencv/android/build.gradle
+++ b/node_modules/react-native-fast-opencv/android/build.gradle
@@ -93,6 +93,7 @@ android {
       cmake {
         arguments "-DANDROID_STL=c++_shared",
-          "-DNODE_MODULES_DIR=${nodeModules}"
+          "-DNODE_MODULES_DIR=${nodeModules}",
+          "-DANDROID_SUPPORT_FLEXIBLE_PAGE_SIZES=ON"
         cppFlags "-O2 -frtti -fexceptions -Wall -fstack-protector-all"
         abiFilters(*reactNativeArchitectures())
       }
@@ -167,7 +168,7 @@ dependencies {
   //noinspection GradleDynamicVersion
   implementation "com.facebook.react:react-android:0.74.4"
   implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-  implementation 'org.opencv:opencv:4.9.0'
+  implementation 'org.opencv:opencv:4.12.0'
 }
 
 if (isNewArchitectureEnabled()) {
